cocos/tween/actions/action-instant.ts

```diff
@@ -253,7 +253,7 @@ export class CallFunc extends ActionInstant {
      * @param {object} [selectorTarget=null]
      * @param {*} [data=null] data for function, it accepts all data types.
      */
-    constructor (selector?: Function, selectorTarget?: any, data?: any) {
+    constructor (selector?: Function | null, selectorTarget?: unknown, data?: any) {
```

Comment:
Why need `null`?

Reply:
In `clone` method, 
```ts
    clone (): CallFunc {
        const action = new CallFunc();
        action.initWithFunction(this._function, this._selectorTarget, this._data);
        return action;
    }
```

initWithFunction access `selector ?: Function`  which means `Function | undefined`, if we don't add `| null`, an error will come out:

```
Argument of type 'Function | null' is not assignable to parameter of type 'Function | undefined'.
  Type 'null' is not assignable to type 'Function | undefined'.ts(2345)
(property) CallFunc._function: Function | null
```

Reply:
We could remove `| null` for constructor here. But could not remove it for `initWithFunction`. The reason is what I said above.